integration-tests/jpa-repository/src/main/resources/application.properties

```diff
@@ -0,0 +1,12 @@
+quarkus.datasource."batch".db-kind=h2
+quarkus.datasource."batch".jdbc.url=jdbc:h2:tcp://localhost/mem:test
+
+quarkus.jberet.repository.type=other
```

Comment:
Hum, we discussed this in the other PR, but looking at it here, it doesn't feel consistent with the rest of the configuration.

Reply:
So what do you suggest ?
Do I remove the enum and use constant strings to identify job repositories ?
(the costraint to have at least 1 injectable job repository at runtime will remain)

Reply:
Yes, please use the string approach with meaningful names. We do need to keep the old enum names for the existing repositories to keep them compatible.

Reply:
Done !